common-lib/src/main/java/com/google/gct/idea/util/PlatformInfo.java

```diff
@@ -5,10 +5,16 @@
 
 import org.jetbrains.annotations.NotNull;
 
+import java.util.Arrays;
+import java.util.List;
+
 public class PlatformInfo {
```

Comment:
we could conceivably make this a service, and then your tests could mock this out instead of changing system props